cfpb/cfgov-refresh
# Upgrade to latest version of cfpb-chart-builder for CCT release

This needs to be deployed Monday 9/24

## Changes

- Upgrade to latest version of CFPB Chart Builder which includes changes from [v3.8.0](https://github.com/cfpb/cfpb-chart-builder/releases/tag/v3.8.0) and [v3.8.1](https://github.com/cfpb/cfpb-chart-builder/releases/tag/v3.8.1)

## Testing

1. `./setup.sh`
1. add a chart block with inquiry data (details in testing notes on #4531)
1. Confirm charts match screenshots 

## Screenshots
Inquiry index
![screen shot 2018-09-21 at 7 28 00 pm](https://user-images.githubusercontent.com/702526/45910108-7f39a200-bdd4-11e8-8c0a-0ddd23873cbf.png)


Credit tightness
![screen shot 2018-09-21 at 7 28 05 pm](https://user-images.githubusercontent.com/702526/45910115-819bfc00-bdd4-11e8-8a29-3c131fd7e582.png)

## Notes

- The charts aren't loading on IE9 and IE10 with the latest version of chart-builder. I've done visual QA in other supported browsers and confirmed it's working in those, so we'll go forward with this version to launch and follow up with thoroughly QAed fixes to IE9/10 after launch.

## Todos

- 

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
